### ISSUE TITLE
"In Combat" flag (and possibly others) resets every relogin

### ISSUE THREAD

**Issue by Arcitec** [Geigerkind/ModifiedPowerAurasTBC#26]
Thanks a lot for porting MPOWA to TBC! There's an annoying bug, though:

Create an aura to show "Battle Shout" (or any other long-term, timed buff), when: "Invert (show icon when aura is missing), Is Alive, In Combat". The point is to show the icon if I don't have that buff while I'm in combat.

But every time I relogin, the aura shows up on screen and when I enter /mpowa I see that "In Combat" has become unchecked. This happens every reload/relogin...

**Comment by Arcitec** [comment]
Update: Every time I log in or /rloadui, I have had to type /mpowa and re-enable the "In Combat" checkbox. I've done that about 20 times in a row over the past few days. So I submitted the ticket.

Now, I've just noticed something weird: I don't have to do that anymore... The addon has properly saved that this aura requires "in combat". What on earth?! I haven't done anything differently. One day (now) it suddenly just worked!
